### DISCUSSION TITLE
Which file controls the size and position of the menu grid, to try and customize the size of sections.

### DISCUSSION THREAD

**Discussion by JunctionRunner** [KlipperScreen/KlipperScreen#1254] [General]
Hi there, so I skimmed through quite a few of the files and didn't spot it for some reason, perhaps it's something that's not easy to do, but I was hoping I might be able to take a crack and customizing klipperscreen a little more than usual, resizing / moving the menu button grid, and expanding some sections like the temperature history graph, or the current print job preview (if you have gcode with the preview from something klipper compatible).

Not a huge issue but it's something that would be fun to give a shot, however I couldn't really see a section that had coordinates for the menu options or what defined the actual layout.

It's entirely possible that it will be something far different than I'm expecting that could cause issues, or just be beyond my ability if it is done in a funky way, but I figure it's more suited as something I'd tinker with, rather than submitting a full feature request.

Also thanks for putting multiple extruders in the top bar after I put a request in, that's freaking awesome! Need to get my printer back up and running and updated to check it out.

**Comment by alfrix** [comment]
margins and paddings base.css icons/image and base text sizes in KlippyGtk and layout directly in the panel colors and other visual effects themes style.css
